CODEOWNERS

```diff
@@ -1041,8 +1041,8 @@ build.json @home-assistant/supervisor
 /tests/components/otbr/ @home-assistant/core
 /homeassistant/components/ourgroceries/ @OnFreund
 /tests/components/ourgroceries/ @OnFreund
-/homeassistant/components/overkiz/ @imicknl @vlebourl @tetienne @nyroDev @tronix117
-/tests/components/overkiz/ @imicknl @vlebourl @tetienne @nyroDev @tronix117
+/homeassistant/components/overkiz/ @imicknl @vlebourl @tetienne @nyroDev @tronix117 @alexfp14
+/tests/components/overkiz/ @imicknl @vlebourl @tetienne @nyroDev @tronix117 @alexfp14
```

Comment:
Thanks! Are you willing to commit contribute to the development and maintenance of the Overkiz integration or would this be a one-off contribution?

Reply:
My idea is to actively contribute to the Overkiz integration, at least to add support for some devices that I have which are not supported yet. I would also be glad to contribute and help maintain the repository. 
However, if you prefer that I am not listed as a code owner yet, that's not a problem. I added myself because I thought it was mandatory.